Header header
geometry_msgs/Quaternion targetLocation
std_msgs/Bool hasTargets
std_msgs/Float32 targetArea
geometry_msgs/Transform cameraPose
std_msgs/Byte pipeline
